### PR TITLE
Components tests

### PR DESCRIPTION
**Pull request by Munyabelden**
### This PR is for

- Added test for all my commponent files

**thank you for reviewing my PR😉**

### CODE REVIEW

File: src/components/__tests__/home.test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import Home from "../home";
+​
+test("renders home page content", () => {
+  render(<Home />);
+​
```
Comment (Zuhaib042) [inline src/components/__tests__/home.test.js]: - [ ] This test is failing. kindly consider adding a `describe` block here. that could be the reason for the failed test.

File: src/components/__tests__/calculator.test.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  expect(headingElement).toBeInTheDocument();
+});
+
+test("adds two numbers correctly", () => {
+  render(<Calculator />);
+  const button1 = screen.getByText("1");
+  const button2 = screen.getByText("2");
+  const addButton = screen.getByText("+");
+  const equalsButton = screen.getByText("=");
+  const result = screen.getByTestId("result");
```
Comment (Zuhaib042) [inline src/components/__tests__/calculator.test.js:17]: - [ ] This line is the reason for another failing test. kindly consider fixing it before submitting for review as we should pass all the tests as per project requirements.

File: src/components/__tests__/calculator.test.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  expect(result).toHaveTextContent("3");
+});
+
+test("subtracts two numbers correctly", () => {
+  render(<Calculator />);
+  const button4 = screen.getByText("4");
+  const button3 = screen.getByText("3");
+  const subtractButton = screen.getByText("-");
+  const equalsButton = screen.getByText("=");
+  const result = screen.getByTestId("result");
```
Comment (Zuhaib042) [inline src/components/__tests__/calculator.test.js:33]: - [ ] This line is the reason for another failing test. kindly consider fixing it before submitting for review as we should pass all the tests as per project requirements.

File: src/components/__tests__/calculator.test.js [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+});
+
+test("multiplies two numbers correctly", () => {
+ render(<Calculator />);
+
+  const button5 = screen.getByText("5");
+  const button7 = screen.getByText("7");
+  const multiply = screen.getByText("x");
+  const equals = screen.getByText("=");
+  const result = screen.getByTestId("result");
```
Comment (Zuhaib042) [inline src/components/__tests__/calculator.test.js:50]: - [ ] This line is the reason for another failing test. kindly consider fixing it before submitting for review as we should pass all the tests as per project requirements.

File: src/components/__tests__/calculator.test.js [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+});
+
+test("divide two numbers correctly", () => {
+  render(< Calculator />);
+
+  const button10 = screen.getByText("8");
+  const button5 = screen.getByText("4");
+  const divide = screen.getByText("÷");
+  const equals = screen.getByText("=");
+  const result = screen.getByTestId("result");
```
Comment (Zuhaib042) [inline src/components/__tests__/calculator.test.js:67]: - [ ] This line is the reason for another failing test. kindly consider fixing it before submitting for review as we should pass all the tests as per project requirements.

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @Munyabelden , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ Professional Readme
✅ No linters errors
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by didierganthier — APPROVED** [review]
Hi @Munyabelden  and @sumon766 ,

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Tests passing✔️ 

## Optional Suggestions 💡
- [ ] Well done overall 💯  but also consider making usage of Jest snapshots for testing all the components. 🤓 

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Once again, congratulations on completing this project! 🎉

**Pull request merged**